data-processing/bot.py

```
@@ -0,0 +1,17 @@
+# bot.py  (only on_message handler changed)
+import json, discord, os
+from t5_router import T5CommandRouter
+# ... (intents + client setup stays the same)
+
+router = T5CommandRouter()
+
+@client.event
```

Ruff (0.8.2)

Undefined name `client`

(F821)